### PR TITLE
Mixed type exercise

### PR DESCRIPTION
**Pull request by AydinHassan**
Requires https://github.com/php-school/php-workshop/pull/226

### CODE REVIEW

File: exercises/all-mixed-up/problem/problem.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+As alluded to earlier in `Unit The Types` PHP's type system has been constantly and consistently growing for a long time.
```
Comment (mikeymike) [inline exercises/all-mixed-up/problem/problem.md]: unite*

Comment (AydinHassan) [reply exercises/all-mixed-up/problem/problem.md]: fixed

**Comment by mikeymike** [comment]
I assume this needs rebasing now? Looks good otherwise

**Comment by AydinHassan** [comment]
> I assume this needs rebasing now? Looks good otherwise 

Yep sorted

**Pull request merged**